ui/src/pages/species/species-columns.tsx

```diff
@@ -77,7 +77,7 @@ export const columns: (projectId: string) => TableColumn<Species>[] = (
       <Link
         to={getAppRoute({
           to: APP_ROUTES.OCCURRENCES({ projectId }),
-          filters: { determination: item.id },
+          filters: { taxon: item.id },
```

Comment:
Just curious, did you notice this had to be updated thanks to build errors? That would be my hope!

Reply:
In this case I was searching the codebase, but I'll run the tests first next time to try that!